e2e/src/usecases/WalletConnectV2.js

```diff
@@ -22,6 +23,11 @@ const dappName = 'WalletConnectV2 E2E'
 const kitUrl = process.env.FORNO_URL || 'https://alfajores-forno.celo-testnet.org'
 const kit = newKit(kitUrl)
 
+const celoClient = createPublicClient({
+  chain: celo,
```

Comment:
I wonder if chain here should indeed be `celo`? Because the testcases are using Alfajores chain id `44787`.

Reply:
Yes you're right technically it should be `celoAlfajores`. Though for the current usage it doesn't matter, I'm changing it to avoid surprises.